### DISCUSSION TITLE
Welcome to obsidian-canvas2document Discussions!

### DISCUSSION THREAD

**Discussion by slnsys** [slnsys/obsidian-canvas2document#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by shenian69** [comment]
Thank you for the plugin. 
I use layout with main .canvas file, that includes multiple inserted .canvas files (like dashboard). 
By now the plugin does not support converting that kind of structure. 

Is it possible to add converting inserted .canvas for your plugin?

**Reply by slnsys** [reply]
Thank you for that input. Embedding of canvases in the output document is something on the roadmap for the next release.
